CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributors Guide
+
+
+### dfmt follows the neptune versioning scheme 
+
+The following is taken verbatim from
+https://github.com/sociomantic-tsunami/neptune/blob/master/doc/library-contributor.rst
+
+
+## Submitting Pull Requests
+
+When making a PR to a Neptune-versioned library, the choice of which branch to
+submit a PR against is important. A branch should be chosen as follows:
+
+* If your change is a bug fix, the PR should be submitted against the oldest
+  supported minor branch.
+* If your change is a new feature, refactoring, or deprecation, the PR should be
+  submitted against the oldest supported major branch.
```

Comment:
What process does a person follow to determine what the "oldest supported major branch" is?

Reply:
He'd go trough the release emails .... which we don't have :)

Reply:
I think we need to introduce at least some parts of this doc too: https://github.com/sociomantic-tsunami/neptune/blob/master/doc/library-user.rst#support-guarantees

The idea is that a project defines a [support guarantee](https://github.com/sociomantic-tsunami/neptune/blob/master/doc/library-user.rst#support-guarantees). I think we are missing specifications for v0.x.x project, which for general semver is pure wilderness but usually we use it respecting the patch release numbering and the minor becomes a mix of major and minor (depending on the projects maturity there might not be patch releases for v0, but we usually do). I think we might need to improve docs in this respect.